ui/conf/nginx.conf

```diff
@@ -2,8 +2,24 @@ server {
     listen       80;
     server_name  localhost;
 
-    #charset koi8-r;
-    #access_log  /var/log/nginx/host.access.log  main;
+    # https://www.digitalocean.com/community/tutorials/how-to-increase-pagespeed-score-by-changing-your-nginx-configuration-on-ubuntu-16-04
```

Comment:
I would rather link the official nginx documentation here: https://nginx.org/en/docs/http/ngx_http_gzip_module.html instead of a "random" blog.